light_node/src/main.rs

```diff
@@ -242,7 +245,63 @@ fn block_on_actors(
     });
 }
 
+unsafe fn enable_core_dumps() {
```

Comment:
@michal-stanek 
could you pls. add all these stuff to the new file light_node\src\system.rs?
for example: in case we want to reuse, we just move this file elsewhere

Reply:
Done